src/scripts/destinationList.js

```diff
@@ -173,72 +196,94 @@ let destinationsList = [
 
   //Wishlist
 
-  {
+  paris123: {
+    id: "paris123",
     city: "Paris",
     country: "France",
     photo: require("../img/paris.jpg"),
     visited: false,
   },
-  {
+  santorini123: {
+    id: "santorini123",
     city: "Santorini",
     country: "Greece",
     photo: require("../img/santorini.jpg"),
     visited: false,
   },
-  {
+  newyork123: {
+    id: "newyork123",
     city: "New York",
     country: "U.S.",
     photo: require("../img/newyork.jpg"),
     visited: false,
   },
-  {
+  marrakech: {
+    id: "marrakech",
     city: "Marrakech",
     country: "Morocco",
     photo: require("../img/marrakech.jpg"),
     visited: false,
   },
-  {
+  amsterdam123: {
+    id: "amsterdam123",
     city: "Amsterdam",
     country: "Netherlands",
     photo: require("../img/Amsterdam.png"),
     visited: false,
   },
-  {
+  berlin123: {
+    id: "berlin123",
     city: "Berlin",
     country: "Germany",
     photo: require("../img/berlin.jpg"),
     visited: false,
   },
-  {
+  lisbon123: {
+    id: "lisbon123",
     city: "Lisbon",
     country: "Porugal",
     photo: require("../img/lisbon.jpg"),
     visited: false,
   },
-  {
+  tokyo123: {
+    id: "tokyo123",
     city: "Tokyo",
     country: "Japan",
     photo: require("../img/tokyo.jpg"),
     visited: false,
   },
-  {
+  sienna123: {
+    id: "sienna123",
     city: "Sienna",
     country: "Italy",
     photo: require("../img/sienna.jpg"),
     visited: false,
   },
-  {
+  dublin123: {
+    id: "dublin123",
     city: "Dublin",
     country: "Ireland",
     photo: require("../img/dublin.jpg"),
     visited: false,
   },
-  {
+  petersburg123: {
+    id: "petersburg123",
     city: "Saint petersburg",
     country: "Russia",
     photo: require("../img/saintpetersburg.jpg"),
     visited: false,
   },
-];
+};
+
+const destinationsList = Object.keys(destinationsObj).map((id) => {
+  destinationsObj[id];
+  // console.log(destinationsObj[id]);
```

Comment:
Here's it's missing the return. In a map, you must always return something.

```return destinationsObj[id];```

Reply:
I couldn't tell what was wrong, it looked like it should work. Your code works without the return so that didn't help :D

Reply:
My code uses an arrow function with the direct return, that's why it doesn't need the word `return`. :) That's the diff

```
const destinationsList = Object.keys(destinationsObj).map(id => destinationsObj[id];
```

Reply:
I try not to copy but do things myself, I'll make sure to copy your code first next time something doesn't work 🙈

Reply:
or use a diff checked 😄